src/components/v2/appDetails/appDetails.store.ts

```diff
@@ -0,0 +1,138 @@
+    },
+    initAppDetailsTabs: (_url: string, displayLogAnalyzer: boolean, isLogAnalyserURL: boolean) => {
+        let aots = [] as Array<ApplicationObject>
+
+        aots.push(addAOT(AppDetailsTabs.k8s_Resources, _url + "/" + URLS.APP_DETAILS_K8, !isLogAnalyserURL))
+
+        if (displayLogAnalyzer) {
+            aots.push(addAOT(AppDetailsTabs.log_analyzer, _url + "/" + URLS.APP_DETAILS_LOG, isLogAnalyserURL))
+            _maxTabAllowd = 7
+        }
+
+        applicationObjectTabsSubject.next([...aots])
+
+        _nodeTreeActiveParentNode = undefined 
+        _nodeTreeActiveNode = undefined
+    },
+    addAppDetailsTab: (tabKind: string, tabName: string, tabURL: string) => {
+
+        if (!tabName || !tabURL || !tabKind) return
+
+        let applicationObjectTabs = applicationObjectTabsSubject.getValue()
+
+        if (applicationObjectTabs.length === _maxTabAllowd) {
+            return false
+        }
+
+        let alredyAdded = false
+        let title = tabKind + '/' + tabName
+        tabName = tabKind + '/...' + tabName.slice(-6)
+
+        for (let index = 0; index < applicationObjectTabs.length; index++) {
+            const tab = applicationObjectTabs[index];
+            tab.isSelected = false
+            if (tab.name.toLowerCase() === tabName.toLowerCase()) {
+                tab.isSelected = true
+                alredyAdded = true
+            }
+        }
+
+        if (!alredyAdded) {
+            applicationObjectTabs.push(addAOT(tabName, tabURL, true, title))
+        }
+
+        applicationObjectTabsSubject.next([...applicationObjectTabs])
+
+        return true;
+    },
+    removeAppDetailsTab: (tabUrl: string): string => {
+        let _applicationObjectTabs = [];
+
+        const applicationObjectTabs = applicationObjectTabsSubject.getValue();
+        let pushURL = '';
+        const pathname = window.location.pathname;
+
+        for (let index = 0; index < applicationObjectTabs.length; index++) {
+            const tab = applicationObjectTabs[index];
+            tab.isSelected = pathname === tab.url || (applicationObjectTabs.length <= 2 && index === 0);
+            if (tab.url !== tabUrl) {
+                _applicationObjectTabs.push(tab);
+
+                if (!pushURL) {
+                    pushURL = pathname === tab.url ? tab.url : '';
+                }
+            }
+        }
+
+        applicationObjectTabsSubject.next([..._applicationObjectTabs]);
+        return pushURL;
+    },
+    markAppDetailsTabActive: (url: string, parentUrl?: string) => {
+        let idTabFound = false
+
+        const applicationObjectTabs = applicationObjectTabsSubject.getValue()
+
+        for (let index = 0; index < applicationObjectTabs.length; index++) {
+            const tab = applicationObjectTabs[index];
+            tab.isSelected = false
+            if (tab.url === url) {
+                tab.isSelected = true
+                idTabFound = true
+            }else if(tab.url.indexOf(parentUrl) > -1){
```

Comment:
this can result in issue if similar type of objects hai similar name for eg http://localhost:3000/chart-store/deployments/82/env/1/details/k8s-resources/clusterrolebinding/keda-operator/manifest and http://localhost:3000/chart-store/deployments/82/env/1/details/k8s-resources/clusterrolebinding/keda-operator-system-auth-delegator/manifest